hrui/CRM/HRUI/Upgrader/Steps/4702.php

```diff
@@ -30,31 +33,6 @@ private function up4702_setCustomCSSURL() {
   }
 
   /**
-   * Sorts Individual Prefixes alphabetically
-   */
-  private function up4702_sortIndividualPrefixes() {
-    // fetch all prefixes sorted alphabetically ( by their labels )
-    // hence ('sort' => 'label asc').
-    $prefixes = civicrm_api3('OptionValue', 'get', [
-      'sequential' => 1,
-      'return' => ['id'],
-      'option_group_id' => 'individual_prefix',
-      'options' => ['limit' => 0, 'sort' => 'label asc']
-    ]);
-
-    // update prefix weights
-    $weight = 1;
-    if (!empty($prefixes['values'])) {
-      foreach($prefixes['values'] as $prefix) {
-        civicrm_api3('OptionValue', 'create', [
-          'id' => $prefix['id'],
-          'weight' => $weight++
-        ]);
-      }
-    }
-  }
-
-  /**
    * Moves Skype option value to the top of IM list
    */
   private function up4702_moveSkypeTop() {

```

Comment:
are you sure that this wouldn't make a duplicate IM , create only update when the ID is provided so you have to fetch the ID for skype option value first (you can use API chaining)